README.md

```diff
@@ -19,22 +20,42 @@ The [Protocol Buffers](https://protobuf.dev) `protoc` command is used to auto-ge
  * Ubuntu/Debian: `sudo apt-get install -y protobuf`
  * Or download pre-compiled binaries: https://github.com/protocolbuffers/protobuf/releases
 
-`protoc` is able to read `.proto` files and generate the message parsers in multiple languages, including Ruby (using the `--ruby_out` option). It does not generate Twirp services and clients; that is where our plugin comes in.
+`protoc` is able to read `.proto` files and generate the message parsers in multiple languages, including Ruby (using
+the `--ruby_out` option). It does not generate Twirp services and clients; that is where our plugin comes in.
 
-### Install the `protoc-gen-twirp_ruby ` plugin
+### Install the `protoc-gen-twirp_ruby` plugin
+ 
+Install the gem by adding it to your Gemfile:
 
-Install the gem and add to the application's Gemfile by executing:
-
-```bash
-bundle add protoc-gen-twirp_ruby --group "development, test"
+```ruby
+group :development, :test do
+  "protoc-gen-twirp_ruby"
+end
 ````
 
-If bundler is not being used to manage dependencies, install the gem by executing:
+Alternatively, install the gem on your system:
```

Comment:
I went with a simplified version of this in twirp-rails: 

https://github.com/collectiveidea/twirp-rails/pull/12/files#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5L16-L23

IMHO, people should know how to install gems by now. 🤷